reflex/components/radix/primitives/base.py

```diff
@@ -26,7 +26,7 @@ def _render(self) -> Tag:
             ._render()
             .add_props(
                 **{
-                    "class_name": format.to_title_case(self.tag or ""),
+                    "class_name": f"{format.to_title_case(self.tag or '')} {self.class_name}",
```

Comment:
Doesn't `class_name` default to `None`? Does this render correctly in that case or does it add an extra argument?